apps/rod/sections/summary-data-sections.js

```diff
@@ -96,16 +96,27 @@ module.exports = {
     {
       field: 'document-type',
       parse: (value, req) => {
-        const yourDocuments = Array.isArray(value) ?
-          value.map(option => option === 'Other' ? req.sessionModel.get('enter-document-type') : option).join(', ') :
-          value;
+        if (!value || (Array.isArray(value) && value.length === 0)) {
+          req.sessionModel.set('yourDocuments', 'Not provided');
+          return 'Not provided';
+        }
+        const yourDocuments = Array.isArray(value)
+          ? value.map(option =>
+            option === 'Other'
+              ? req.sessionModel.get('enter-document-type') || 'Other (not specified)'
+              : option
+          ).join(', ')
+          : value;
         req.sessionModel.set('yourDocuments', yourDocuments);
         return yourDocuments;
       }
     },
     {
       step: '/your-documents',
-      field: 'document-description'
+      field: 'document-description',
+      parse: value => {
+        return value && value.trim() ? value : 'Not provided';
```

Comment:
I would keep the default string in a variable so

- easy to change if required in the future
- less chances of typing it wrong because its used for multiple fields